### PR TITLE
card component

### PR DESCRIPTION
**Pull request by akash-1318**

### CODE REVIEW

File: components/Card.html [hunk 1]
```diff
@@ -36,7 +36,293 @@
                 </ul>
             </aside>
             <div class="main__grid-item">
-                
+                <div class="component">
+                    <p class="component__heading">Cards</p>
+                    <p class="component__text">Cards are built with as little markup and styles as 
+                        possible, but still manage to deliver a ton of control and customization.</p>
```
Comment (viru2001) [inline components/Card.html:42]: you can properly format this

File: components/Card.html [hunk 2]
```diff
@@ -36,7 +36,293 @@
...
+                                </div>
+                            </footer>
+                        </div>
+                    </div> 
+                    <div class="code__snippet">
+                        <iframe
+                        src="https://carbon.now.sh/embed/xiZGIfECKaumC6IqpozC"
+                        style="width: 808px; height: 834px; border:0; transform: scale(1); overflow:hidden;"
+                        sandbox="allow-scripts allow-same-origin">
+                        </iframe>
```
Comment (viru2001) [inline components/Card.html]: avoid using inline styles. you can make class for code snippets styling

File: components/css/componentStyle/Card.css [hunk 3]
```diff
@@ -0,0 +1,226 @@
+.card{
+    width: 30rem;
+    border-radius: 5px;
+    overflow: hidden;
+    box-shadow: 5px 5px 10px #00000038;
```
Comment (viru2001) [inline components/css/componentStyle/Card.css:5]: use variable for color

File: components/css/componentStyle/Card.css [hunk 4]
```diff
@@ -0,0 +1,226 @@
...
+    cursor: pointer;
+    /* display: flex;
+    flex-direction: column; */
```
Comment (viru2001) [inline components/css/componentStyle/Card.css]: you can remove commented code

File: components/css/componentStyle/Card.css [hunk 5]
```diff
@@ -0,0 +1,226 @@
...
+    position: relative;
+}
+
+.card__container{
+    display: flex;
+    flex-direction: column;
+}
+
+.card__name{
+    padding: 0rem 2rem 0rem 2rem;
```
Comment (viru2001) [inline components/css/componentStyle/Card.css]: you can use this here
```css
padding : 0rem 2rem;
```

**Review by viru2001 — COMMENTED** [review]
I have given few suggestions.
make sure to format your HTML code and remove commented code from CSS.

File: components/Card.html [hunk 6]
```diff
@@ -36,7 +36,293 @@
...
+                                <div class="card-footer-right-div">
+                                <span><i class='bx bx-heart'></i></span> 
+                                <span><i class='bx bx-share-alt'></i></span> 
+                                <span><i class='bx bx-dots-vertical-rounded'></i></span> 
+                                </div>
+                            </footer>
+                        </div>
+                    </div> 
+                    <div class="code__snippet">
+                        <iframe
```
Comment (yogesh-) [inline components/Card.html:118]: Avoid inline styling. For snippets make 1 class and use it everywhere.

File: components/css/componentStyle/Card.css [hunk 7]
```diff
@@ -0,0 +1,226 @@
...
+    font-size: 1.2rem;
+    cursor: pointer;
+    text-align: center;
+    padding: 1rem;
+}
+
+/* .card-footer p:hover{
+    background-color: rgb(248, 240, 255);
+    border-radius: .5rem;
+} */
```
Comment (yogesh-) [inline components/css/componentStyle/Card.css]: remove commented code

**Review by yogesh- — CHANGES_REQUESTED** [review]
few minor changes suggested, rest all looks good

**Pull request merged**